tutorial/source/normalizing_flows_intro.ipynb

```diff
@@ -4,10 +4,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Normalizing Flows - Introduction (Part 1)\n",
+    "# Normalizing Flows - Introduction\n",
+    "\n",
+    "This tutorial introduces Pyro's built-in normalizing flows. It is independent of most of Pyro, but users may want to read about distribution shapes in the [Tensor Shapes Tutorial](http://pyro.ai/examples/tensor_shapes.html).\n",
```

Comment:
can you add something like this to the top of both example notebooks?

"In this notebook we leverage normalizing flows implemented in Zuko, but similar functionality could also be obtained with another PyTorch-based normalizing library."